### PR TITLE
Add FPM and Alpine variant

### PR DESCRIPTION
**Pull request by J0WI**
Fixes https://phabricator.wikimedia.org/T197684

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -15,8 +15,20 @@ jobs:
     steps:
     - uses: actions/checkout@v1
     - name: v1.31
```
[the remark this answers is not in the document]
Comment (J0WI) [reply .github/workflows/build.yml]: done

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -14,9 +14,27 @@ jobs:
...
+    - name: v1.34-fpm
+      run: docker build ./1.34/fpm
+    - name: v1.34-fpm-alpine
+      run: docker build ./1.34/fpm-alpine
+    - name: v1.35-apache
+      run: docker build ./1.35/apache
+    - name: v1.35-fpm
+      run: docker build ./1.35/fpm
+    - name: v1.35-fpm-alpine
+      run: docker build ./1.34/fpm-alpine
```
Comment (davidbarratt) [inline .github/workflows/build.yml:40]: I think this should be 1.35 ?

Comment (J0WI) [inline .github/workflows/build.yml:40]:
```suggestion
      run: docker build ./1.35/fpm-alpine
```

Comment (J0WI) [reply .github/workflows/build.yml]: Indeed, thanks!

**Comment by davidbarratt** [comment]
@J0WI You're failing and it's not my fault this time! (I don't think, ha)

**Comment by J0WI** [comment]
Flaky GnuPG verification, let's just try again.

**Comment by davidbarratt** [comment]
Third time was a charm:
https://github.com/wikimedia/mediawiki-docker/pull/81/checks?check_run_id=971844062

**Comment by edvinkuric** [comment]
Hey guys,

Thank you for the pull request.
Is there any official tag on docker-hub to use the alpine-variant of this image?

**Comment by davidbarratt** [comment]
> Is there any official tag on docker-hub to use the alpine-variant of this image?

Not yet. Would you mind opening a PR to update this file:
https://github.com/docker-library/official-images/blob/master/library/mediawiki
?

If not, I can whenever 1.35 is released. :smile:

**Pull request merged**